app/renderer/components/navigation/urlBar.js

```diff
@@ -125,11 +118,10 @@ class UrlBar extends React.Component {
           const isLocationUrl = isUrl(location)
           if (!isLocationUrl && e.ctrlKey) {
             appActions.loadURLRequested(this.props.activeTabId, `www.${location}.com`)
-          } else if (this.shouldRenderUrlBarSuggestions &&
-              ((typeof this.activeIndex === 'number' && this.activeIndex >= 0) ||
+          } else if (this.props.showrUrlBarSuggestions &&
+              ((typeof this.props.activeIndex === 'number' && this.props.activeIndex >= 0) ||
               (this.props.urlbarLocationSuffix && this.props.autocompleteEnabled))) {
             // Hack to make alt enter open a new tab for url bar suggestions when hitting enter on them.
-            const isDarwin = process.platform === 'darwin'
             if (e.altKey) {
               if (isDarwin) {
```

Comment:
Using this utility method is encouraged over duplicating the code (especially since it has tests 😄 )